### ISSUE TITLE
Dashboard\Admin\Contacts - External User has conditional tile link display

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#1557]
#### Description:

The "Contacts Management" tile should only appear on the main dashboard under specific circumstances.
[Figma link](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7702-4102&mode=design&t=urN0lS2jx1k3p5Ve-4)

#### Acceptance Criteria:

Given I am an authenticated user with the `industry-user-admin` role (i.e., I have been Approved to have `admin` rights for an Operator)
When I navigate to the /dashboard page or /administration page
Then the "Contacts" tile displays in the dashboard

Given I am an authenticated user with the `industry-user` role but have not been Approved to have access to any Operator
When I navigate to the /dashboard page or /administration page
Then the "Contacts" tile DOES NOT display in the dashboard

Given I am an authenticated user with the `industry-user` role and I have been Approved to have access to an Operator
When I navigate to the /dashboard page or /administration page
Then the "Contacts" tile displays in the dashboard

#### Development Checklist:

- [ ] In `bc_obps/common/fixtures/dashboard/bciers/external.json` add conditionally objects for `Contacts` links, similar to "Select an Operator"; "My Operator"
- [ ] Contacts tile renders conditionally based on user roles
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by

**Comment by andrea-williams** [comment]
@nanyangpro please confirm desired behaviour in Acceptance Criteria

**Comment by patrickisaac** [comment]
Will continue discussion next backlog refinement

**Comment by shon-button** [comment]
@patriciarussellCAS 

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/dashboard
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration

LOGINS;
`bc-cas-dev`: "Contacts" available
`bc-cas-dev-secondary`:  "Contacts" NOT available

**Comment by patriciarussellCAS** [comment]
Reviewed in Dev. 
Notes: 
- Unable to confirm the third scenario where a 'industry-user' has been given permission and can now see contacts. Otherwise this works as intended. Perhaps someone can quickly walk through that process with me?

- Question: If we are blocking the view of "contacts" for secondary users, should we also be blocking "users and access requests"? I'm unclear why one is blocked and not the other.
